catroid/src/main/res/values/strings.xml

```diff
@@ -901,6 +901,8 @@
     <string name="show_password">Show Password</string>
     <string name="register_password_mismatch">Passwords do not match</string>
     <string name="sign_in_error">Error during Sign-In</string>
+    <string name="error_connecting_to">Error: connecting failed to %s : %d%%</string>
+    <string name="error_google_plus_sign_in">ERROR Google+ Sign in. Status: %s</string>
```

Comment:
Maybe you have some example status messages (since you used `result.getStatus().getStatusMessage()`)?